src/it/scala/org/ergoplatform/it/api/NodeApi.scala

```diff
@@ -150,4 +189,14 @@ object NodeApi extends ScorexLogging {
 
   case class Status(status: String)
 
+  case class HistoryInfo(bestHeaderId: String, bestBlockId: String, bestHeaderHeight: Int, bestBlockHeight: Int)
```

Comment:
we already have `ErgoStatsCollector#NodeInfo`, better to reuse it

Reply:
@catena2w, `ErgoStatsCollector#NodeInfo` contains too much redundant data, that should be fetched not only from API, but from settings as well, so its not conveniently to use it here.